Makefile

```diff
@@ -8,53 +8,33 @@ build-builder:
 		--build-arg UID=$(shell id -u) --build-arg GID=$(shell id -g) \
 		ci
 
-.PHONY: build-node
-build-node:
-	docker build --tag aleph-onenode-chain --file ci/Dockerfile.aleph-node ci
-
-.PHONY: run-node
-run-node: build-node # Run a one-node chain in docker.
-	docker run --detach --rm --network host \
-		--name ink-wrapper-node \
-		aleph-onenode-chain
-
 .PHONY: test_contract
 test_contract:
-	cd test-project/test_contract && cargo contract build --release
-
-.PHONY: upload-test-contract
-upload-test-contract: test_contract
-	cd test-project/test_contract && cargo contract upload --suri //Alice --url ws://localhost:9944 || true
+	cd tests/test_contract && cargo contract build --release
 
 .PHONY: psp22_contract
 psp22_contract:
-	cd test-project/psp22_contract && cargo contract build --release
-
-.PHONY: upload-psp22-contract
-upload-psp22-contract: psp22_contract
-	cd test-project/psp22_contract && cargo contract upload --suri //Alice --url ws://localhost:9944 || true
+	cd tests/psp22_contract && cargo contract build --release
 
 .PHONY: test_contract.rs
 test_contract.rs: test_contract
 	cd ink-wrapper && \
-		cargo run -- -m ../test-project/test_contract/target/ink/test_contract.json \
-			--wasm-path ../test_contract/target/ink/test_contract.wasm \
-		| rustfmt --edition 2021 > ../test-project/src/test_contract.rs
+		cargo run -- -m ../tests/test_contract/target/ink/test_contract.json \
+			--wasm-path ../../test_contract/target/ink/test_contract.wasm \
+		| rustfmt --edition 2021 > ../tests/drink/src/test_contract.rs
 
 .PHONY: psp22_contract.rs
```

Comment:
should it be phony?

Reply:
What do you mean? From my understanding - if the target isn't the file produced then it's `.PHONY`. Am I missing something?

Reply:
Yeah, you could also change the target to `tests/drink/src/psp22_contract.rs`. Although then timestamp checking does kick in, and you have to make sure that the file is properly rebuilt when its dependencies change. I think `.PHONY` is just easier if it's fast, and then let `cargo` worry about what needs to be rebuilt on the lower levels.